mitodl/mitxonline
# Home Page Contact Component

# What are the relevant tickets?
updates https://github.com/mitodl/hq/issues/1677
closes https://github.com/mitodl/hq/issues/2166

# Description (What does it do?)
Adds the hubspot form and social links to the bottom of the home page.

# Screenshots (if appropriate):
![Screenshot from 2023-09-05 11-17-22](https://github.com/mitodl/mitxonline/assets/7756053/68738019-0dea-4ce6-98b0-e893e1b28cd7)
![Screenshot from 2023-09-05 10-34-34](https://github.com/mitodl/mitxonline/assets/7756053/5ecdc7bb-708e-4eaa-ad00-72aced8ea937)


# How can this be tested?
Load everything, get the values for HUBSPOT_HOME_PAGE_FORM_GUID and HUBSPOT_PORTAL_ID from the linked [issue](https://github.com/mitodl/hq/issues/2166).
rebuild your container and load the home page - you should see the contact component. Test it at different widths and test that you can submit the form and that the resulting text is looking ok
Figma attached to check styling

# Additional Context
[Figma](https://www.figma.com/file/Gs4zIhOFv5gVvafawwl6PF/MITx-Online?node-id=929%3A5249&mode=dev)
